content/ko/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -134,13 +134,13 @@ CloudStack 클라우드 제공자는 쿠버네티스 노드 오브젝트의 이
 GCE 클라우드 제공자는 쿠버네티스 노드 오브젝트의 이름으로 노드의 (kubelet에 의해 결정되거나 `--hostname-override` 로 재정의된) 호스트 이름을 사용한다.
 참고로 쿠버네티스 노드 이름의 첫 번째 세그먼트는 GCE 인스턴스 이름과 일치해야 한다(예: `kubernetes-node-2.c.my-proj.internal` 이름이 지정된 노드는 `kubernetes-node-2` 이름이 지정된 인스턴스에 해당해야 함).
 
-## HUAWEI 클라우드
+## Huawei 클라우드
```

Comment:
Before updating the Korean localization, fix the English original on the master branch. Although I believe this text should read “HUAWEI CLOUD” as that is the product name.

Reply:
Changed it back. Thank you!